libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -401,7 +401,7 @@ void CodeTransform::operator()(FunctionDefinition const& _function)
 	yulAssert(m_scope->identifiers.count(_function.name), "");
 	Scope::Function& function = std::get<Scope::Function>(m_scope->identifiers.at(_function.name));
 
-	int height = m_evm15 ? 0 : 1;
+	size_t height = m_evm15 ? 0 : 1;
```

Comment:
@mijovic why is this first changed to `size_t` and then cast back to `int`?

Reply:
I see that it's assigned to both `int`s and `size_t`s below so it would need casts either way. I think making it `size_t` and casting to `int` only when necessary makes sense since it should not be negative.